src/genie/libs/parser/junos/show_interface.py

```diff
@@ -1480,6 +1480,7 @@ def cli(self, interface=None, output=None):
         lag_int_type = ''
         lacp_flag = ''
         if_dist_dict = {}
+        address_family_dict = {}
```

Comment:
Good catch

I would recommend also changing line 1938 to use `.update` like on line 1946 so that we don't overwrite the dictionary; only add to it/update it
![image](https://user-images.githubusercontent.com/33130026/132025955-cf6d5bef-ae61-4e4e-810e-817ea25d72a1.png)

Reply:
@LukasMcClelland I hope it's fine now